bundles/org.openhab.binding.verisure/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -160,6 +160,7 @@
 			<channel id="installationName" typeId="installationName"/>
 			<channel id="installationId" typeId="installationId"/>
 			<channel id="smokeDetectorTriggerChannel" typeId="triggerChannel"/>
+			<channel id="lowBattery" typeId="lowBattery"/>
```

Comment:
There is no need to introduce a new channel type. We have a [system default channel type](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) for `low-battery`

```suggestion
			<channel id="lowBattery" typeId="system.low-battery"/>
```

Reply:
Fixed!